src/components/SignerPanel/SigningStatus.js

```diff
@@ -0,0 +1,290 @@
+import React, { useMemo } from 'react'
```

Comment:
I see a lot of language here that's intended only for signing transactions, but what about the case where we have to make an API call, like in committing votes?